src/main/resources/templates/email/user/addedToTerraware/body.ftlh.mjml

```diff
@@ -3,18 +3,49 @@
     <mj-include path="../../head.ftlh.mjml"/>
 
     <mj-body>
-        <mj-include path="../../logo.ftlh.mjml"/>
+        <mj-section padding="24px 0px" background-color="#F9F8F7">
+            <mj-column padding="0">
+                <mj-image mj-class="logo-img" src="${webAppUrl}/assets/tw-tf-logo.png">
+                </mj-image>
+            </mj-column>
+            <mj-column padding="0">
+                <mj-text mj-class="text-header01">
+                    ${strings("notification.user.addedToOrganization.email.header.1")}</mj-text>
```

Comment:
Aside from things like the page footer and button labels, we've generally erred on the side of having a separate set of strings for each notification type even if some of them are duplicates of other notifications. That way we're able to tweak the wording on just one of the notifications without accidentally affecting others, and it's easier to look at the messages file and see the contents of a notification. Mixing references to different notifications means you have to go back and forth between the MJML file and the strings file to see what's in a message.

Which is to say: I think we should have all the strings for this notification under `notification.user.addedToTerraware` (aside from shared components like the "Terraware benefits" list and footers/button labels) even though some of them are the same as the "added to organization" strings.